backend/support/cog_over.py

```diff
@@ -45,13 +49,24 @@ def compute_cog_over(federal_awards, submission_status, auditee_ein, auditee_uei
         oversight_agency = agency
         # logger.warning("Assigning an oversight agenct", oversight_agency)
         return (cognizant_agency, oversight_agency)
-    cognizant_agency = determine_hist_agency(auditee_ein, auditee_uei)
+    cognizant_agency = determine_hist_agency(auditee_ein, auditee_uei, base_year)
     if cognizant_agency:
         return (cognizant_agency, oversight_agency)
     cognizant_agency = agency
     return (cognizant_agency, oversight_agency)
 
 
+def calc_base_year(audit_year):
+    # Note: 2019 is the first supported baseline year in GSAFAC
+    # For audit years 2019 through 2023, baseline year is 2019
+    # For audit years 2024 through 2028, baseline year is 2024
+    # For audit years 2029 through 2033, baseline year is 2029
+    # For audit years 2034 through 2038, baseline year is 2034
+    # and so on
+    base_year = (math.floor((int(audit_year) - 2019) / 5) * 5) + 2019
```

Comment:
2019 and 5 appear to be magic numbers. I would prefer if pulled these out into constants. (2019 is also used below in line 97)